src/construction/construction_helpers.py

```diff
@@ -149,3 +149,29 @@ def clean_construction_type(value: str) -> str:
     # remove whitespaces
     cleaned = "_".join(cleaned.split())
     return value
+
+
+def remove_short_to_long_0(updated_snapshot_df, construction_df):
+    """Remove instance 0 for short to long constructions
+
+    Args:
+        updated_snapshot_df (pd.DataFrame): The updated snapshot df.
+        construction_df (pd.DataFrame): The construction df.
+
+    Returns:
+        pd.DataFrame: The updated snapshot df with instance 0
+        removed for short to long constructions.
+    """
+    short_to_long_references = construction_df.loc[
+        construction_df["construction_type"].str.lower() == "short_to_long",
```

Comment:
construction type should already be cleaned here, so `construction_df["construction_type"]=="short_to_long"` should work fine here (less operations e.g., converting to a string and then lowering)